docs/reference/viewsets.md

```diff
@@ -64,7 +64,7 @@ Viewsets are Wagtail's mechanism for defining a group of related admin views wit
 
    .. attribute:: model
 
-   Required; the model class that this viewset will work with. The :attr:`~django.db.models.Options.model_name` will be used
+   Required; the model class that this viewset will work with. The `~django.db.models.Options.model_name` will be used
```

Comment:
This renders as italics, which I don't think was intended - given that this reference doesn't resolve, I'll change this to a plain `model_name` (in double backticks).